servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/BaseTestNessieApi.java

```diff
@@ -1556,4 +1560,122 @@ public void fetchEntriesByNamelessReference() throws BaseNessieClientServerExcep
             immutableEntry(a, Content.Type.ICEBERG_TABLE),
             immutableEntry(b, Content.Type.ICEBERG_VIEW));
   }
+
+  @Test
+  @NessieApiVersions(versions = NessieApiVersion.V2)
+  public void relativeCommitLocations() throws BaseNessieClientServerException {
+    Branch main = api().getDefaultBranch();
+
+    if (!fullPagingSupport()) {
```

Comment:
nit: `assumeThat(fullPagingSupport()).isTrue()`?

Reply:
Not here until we have Quarkus 3

Reply:
I mean using assumptions instead of plain returning from a test - a failed assumption is reported nicer at the test runner level.

Reply:
done